website/docs/language/import/#

```diff
@@ -0,0 +1,131 @@
+---
+page_title: Import - Configuration Language
```

Comment:
Can we remove this file? Vercel won't build successfully unless every file is reference in the `language-nav-data.json` file, and it doesn't seem like we need this one?